drools-ansible-rulebook-integration-api/src/main/java/org/drools/ansible/rulebook/integration/api/domain/RulesSet.java

```diff
@@ -32,6 +32,8 @@ public class RulesSet {
     private int enabledRulesNumber;
     private int disabledRulesNumber;
 
+    private long delayWarningThreshold = 5000; // Default 5 seconds
```

Comment:
Configurable with ruleSet parameter `delay_warning_threshold`. Default is 5 seconds.